packages/indexer-common/src/indexer-management/types.ts

```diff
@@ -161,6 +161,10 @@ const Caip2ByChainAlias: { [key: string]: string } = {
   hardhat: 'eip155:1337',
   'arbitrum-one': 'eip155:42161',
   'arbitrum-goerli': 'eip155:421613',
+  avalanche: 'eip155:43114',
+  polygon: 'eip155:137',
```

Comment:
Should we also add `matic` for referring to `polygon`?
I'm not sure if that alias is used elsewhere.

Reply:
I think we are going to try to use `polygon` everywhere on the network.